### PR TITLE
Removed support for DB-generated UUIDs

### PR DESCRIPTION
**Pull request by morozov**
|      Q       |   A
|------------- | -----------
| Type         | improvement
| BC Break     | yes
| Fixed issues | #3167

### CODE REVIEW

File: docs/en/reference/sharding.rst [hunk 1]
```diff
@@ -100,11 +100,12 @@ platforms:
 
     <?php
     use Doctrine\DBAL\DriverManager;
+    use Rhumsaa\Uuid\Uuid;
```
Comment (stloyd) [inline docs/en/reference/sharding.rst]: As you suggest to use `ramsey/uuid` library I guess those namespaces should be: `Ramsey\Uuid\Uuid`

**Review by stloyd — CHANGES_REQUESTED** [review]

File: docs/en/reference/sharding.rst (hunk 1 above)
Comment (morozov) [reply docs/en/reference/sharding.rst]: Thank you, @stloyd.

**Review by Ocramius — APPROVED** [review]
LGTM. Waiting for more opinions by @doctrine/doctrinecore before FIRING ZEH MISSILES!

**Review by Majkl578 — COMMENTED** [review]
Needs to be dropped in ORM first. Otherwise :+1:

**Review by malarzm — APPROVED** [review]

**Comment by RusAlex** [comment]
Hi, mates. Sorry it's not any link to issue. Where can I read , why this changes are needed ?

**Comment by Majkl578** [comment]
@RusAlex See #3167.

**Review by Majkl578 — APPROVED** [review]
Ready to go after rebase. :ok_hand:

**Pull request merged**